README.md

```diff
@@ -107,7 +107,27 @@ If at some point you see something that could be better, then please open a PR.
 
 `pacman -S --needed base-devel git arch-install-scripts repoctl fuse-overlayfs rsync python-telegram-send`
 
-One must have an active mirror of chaotic-aur running locally and some signing key. Configure them in `/etc/chaotic.conf`, like this:
+One must have an active mirror of chaotic-aur running locally and some signing key. 
+To create a gpg key & ssh keypair for root user do the following (chowning tty is needed as the process fails otherwise):
```

Comment:
> (chowning tty is needed as the process fails otherwise)

This is actually a hack you needed because of sudo/su, please remove it and leave a mention pointing to https://wiki.archlinux.org/index.php/GnuPG#su